rust/kcl-lib/src/std/revolve.rs

```diff
@@ -127,14 +127,14 @@ async fn inner_revolve(
 
     let mut solids = Vec::new();
     for sketch in &sketches {
-        let id = exec_state.next_uuid();
+        let new_solid_id = exec_state.next_uuid();
```

Comment:
Renaming this isn't related to the problem, but it made the code easier to read so I did it.